vcpkg/concepts/default-features.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Understanding default features in vcpkg
+description: This article explains the concept of default features, their capabilities, and their expected contents in detail.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 02/01/2024
+ms.topic: concept-article
+---
+
+# Concept: Default features
+
+This article delves into the concept of default features, their implications, and strategies for effective management.
+
+Default features in vcpkg are a set of features automatically included when a port is installed, unless explicitly disabled. These features ensure that a library maintains a baseline level of functionality, even as the complexity of its dependency graph increases.
+
+## Role of default features
+
+Default features are crucial for providing a user-friendly experience with libraries that offer modular functionalities through features. For instance, a library capable of extracting various archive formats might have each format as an optional feature. Without default features, the library might not perform any extraction out of the box, complicating initial use. Default features guarantee that essential functionalities are available immediately, enhancing usability.
+
+## Specifying default features
+
+When adding a library as a dependency in your `vcpkg.json` manifest, default features are included by default. To modify this behavior, you can use the `"default-features": false` attribute to disable them, allowing for a more customized setup based on your specific needs.
```

Comment:
I think we should call out the "but only if specified in the top-level manifest" requirement here. Since writing this article was motivated by user confusion of it not working on transitive dependencies.